macros/macro-test/src/main.rs

```diff
@@ -15,6 +15,16 @@ impl Port for PingPongPort {
     type Request = Ping;
 }
 
+// Tests that the macro allows multiple ports
+#[derive(ComponentDefinition, Actor)]
+struct Pinger2 {
+    ctx: ComponentContext<Pinger>,
+    ppp: RequiredPort<PingPongPort>,
+    ppp2: RequiredPort<PingPongPort>,
```

Comment:
Two ports of the same type are not going to work, since it will try to double implement a trait.

Reply:
Go back to your vacation! We noticed the problem as well 😄